### ISSUE TITLE
Error with simd_wasm64 feature

### ISSUE THREAD

**Issue by Titaniumtown** [Craig-Macomber/lol_alloc#1]
I tried using this in my wasm project and I get the following error:
```
error[E0658]: use of unstable library feature 'simd_wasm64'
  --> /home/simon/.cargo/git/checkouts/lol_alloc-22b2a73873fe6f0a/ecf5399/lol_alloc/src/lib.rs:1:1
   |
1  | / #![no_std]
2  | |
3  | | #[cfg(test)]
4  | | #[macro_use]
...  |
35 | |     fn memory_grow(&self, delta: PageCount) -> PageCount {
36 | |         PageCount(core::arch::wasm::memory_grow(0, delta.0))
   | |_______________________________________________^
   |
   = note: see issue #90599 <https://github.com/rust-lang/rust/issues/90599> for more information
   = help: add `#![feature(simd_wasm64)]` to the crate attributes to enable
```

**Comment by Craig-Macomber** [comment]
Odd. What version of rust are you using? I'm using 1.62.0.

What target are you using? I only tested with `wasm32-unknown-unknown`. Are you using the tier 3 [wasm64-unknown-unknown](https://doc.rust-lang.org/nightly/rustc/platform-support/wasm64-unknown-unknown.html) target?

**Comment by Titaniumtown** [comment]
`cargo build --release --target wasm32-unknown-unknown` I am compiling for wasm32, and I'm using the latest nightly build as I need those features.

**Comment by Craig-Macomber** [comment]
I have no idea why calling core::arch::wasm::memory_grow claims to need `#![feature(simd_wasm64)]`. I'm on stable (and thus can even use such features) and it works fine.

Looks like core::arch::wasm is documented as marked with needing simd_wasm64: https://doc.rust-lang.org/core/arch/wasm/index.html

Maybe this is a recent regression on nightly?

**Comment by Craig-Macomber** [comment]
I have published a new version that should work around this issue.

**Comment by Craig-Macomber** [comment]
Looks like `core::arch::wasm::memory_grow` working at on on stable was a rustc bug, and its just an alias for `core::arch::wasm32::memory_grow` so my change is a good fix.

The readme is now more clear about just supporting wasm32 as well. I'm going to consider this resolved, but feel free to re-open if its not fixed for you in `0.1.2`

**Comment by Titaniumtown** [comment]
Thanks!
